previewer/main_renderer.js

```diff
@@ -82,4 +83,13 @@ $(document).ready(function(){
       console.log("unknown objectType: " + asset)
     }
   })
+
+    tagsMap.forEach(function(tagCount, tag) {
+        $('#tagList').append(
+            $('<span/>', {class: 'tag'})
+                .append([
+                    $('<span/>', {class: 'tag-name'}).text(`${tag}`),
+                    $('<span/>', {class: 'tag-count'}).text(`${tagCount}`)
+                ]));
+    })
```

Comment:
Don't fight the framework if there's no need:
https://getbootstrap.com/docs/4.0/components/badge/